CMakeLists.txt

```diff
@@ -129,6 +129,10 @@ if(OPENCV_WORKAROUND_CMAKE_20989)
   set(CMAKE_SYSTEM_PROCESSOR ${CMAKE_SYSTEM_PROCESSOR_BACKUP})
 endif()
 
+if(CMAKE_SYSTEM_NAME MATCHES QNX)
+  set(OPENCV_PYTHON_INSTALL_PATH ${QNX_PYTHON_INSTALL_PATH})
```

Comment:
We don't need to introduce `QNX_PYTHON_INSTALL_PATH` (google doesn't know that anyway).
Just specify `OPENCV_PYTHON_INSTALL_PATH` value to CMake thorugh command-line or via `cmake/OpenCVInstallLayout.cmake`

Reply:
Thanks, I removed this and passed `OPENCV_PYTHON_INSTALL_PATH` via toolchain.